core-bundle/contao/templates/backend/be_popup.html5

```diff
@@ -89,6 +89,22 @@
           </div>
         </div>
       <?php endif; ?>
+      <?php if ($this->metadata): ?>
+        <?php foreach ($this->metadata as $format => $table): ?>
+          <table class="tl_show zebra-table zebra-table--no-border">
+            <caption><?= $this->trans('MSC.imageMetadata', ['%s' => Contao\StringUtil::specialchars(strtoupper('iptc' === $format ? 'IPTC IIM' : $format))]) ?></caption>
+            <?php foreach ($table as $label => $values): ?>
+              <?php if ('' === trim(implode('', $values))) {
+                continue;
+              } ?>
+              <tr>
+                <td><span class="tl_label"><?= Contao\StringUtil::specialchars($label) ?></span></td>
+                <td><?= Contao\StringUtil::specialchars(implode(', ', $values)) ?></td>
+              </tr>
```

Comment:
```suggestion
              <?php if (count(array_filter($values))): ?>
                <tr>
                  <td><span class="tl_label"><?= Contao\StringUtil::specialchars($label) ?></span></td>
                  <td><?= Contao\StringUtil::specialchars(implode(', ', $values)) ?></td>
                </tr>
              <?php endif; ?>
``````

This matches the template coding style a bit more. 🙈

Reply:
`count(array_filter($values, \strlen(...)))` should work I think because the string `'0'` should not be filtered out.